scripts/travis-run.sh

```diff
@@ -1,6 +1,16 @@
 #!/bin/bash
 set -euo pipefail
 
+if [[ $TRAVIS_BRANCH != "master" && $TRAVIS_PULL_REQUEST == "false" && $TRAVIS_REPO_SLUG == "bioconda/bioconda-recipes" ]]
+then
+    echo ""
+    echo "Tests are skipped for pushes to the main bioconda-recipes repo."
+    echo "If you have opened a pull request, please see the full tests for that PR."
+    echo "See https://bioconda.github.io/build-system.html for details"
+    echo ""
+    exit 0
+fi
+
```

Comment:
You are right. Ok, let's do it like this. But additionally, could you add the same check to travis-setup? This way, we also skip the deployment of miniconda, and get a minimal build time.